ruth/vehicle.py

```diff
@@ -25,27 +25,51 @@ def set_numpy_type(name, fld=None):
 IndexedNode = namedtuple("IndexedNode", ["node", "index"])
 
 
-class VehicleBehavior(IntFlag):
+class VehicleAlternatives(IntFlag):
     DEFAULT = 0
     DIJKSTRA_FASTEST = auto()
     DIJKSTRA_SHORTEST = auto()
     PLATEAU_FASTEST = auto()
 
 
-def set_vehicle_behavior(vehicles: List['Vehicle'], ratio: List[float], behavior: List[VehicleBehavior]):
+class VehicleRouteSelection(IntFlag):
+    NO_ALTERNATIVE = 0
+    FIRST = auto()
+    RANDOM = auto()
+    PTDR = auto()
+
+
+def set_vehicle_behavior(vehicles: List['Vehicle'],
```

Comment:
I have no idea what's going on here, could you please add some comment? :) Thanks!